connectors/concur/src/test/java/com/vmware/connectors/concur/ConcurControllerTests.java

```diff
@@ -246,17 +289,18 @@ private void testRequestCards(final String requestFile,
     }
 
     private ResponseActions expect(final String issue) {
-        return this.mockConcur.expect(requestTo("https://concursolutions.com/expense/reports/" + issue))
+        return this.mockConcur.expect(requestTo("https://implementation.concursolutions.com/api/expense/expensereport/v2.0/report/" + issue))
                 .andExpect(method(GET))
-                .andExpect(MockRestRequestMatchers.header(AUTHORIZATION, "Bearer concur-token"));
+                .andExpect(MockRestRequestMatchers.header(AUTHORIZATION, "OAuth 0_xxxxEKPk8cnYlWaos22OpPsLk="))
+                .andExpect(MockRestRequestMatchers.header(ACCEPT, APPLICATION_JSON_VALUE));
```

Comment:
Does the 2.0 endpoint really support JSON?  I only see XML mentioned in the docs: https://developer.concur.com/api-reference/expense/expense-report/expense-report-get.html

Reply:
Yes, it supports json. By default, it will xml reponse. We have to add "Accept - application/json" in the request header to receive the json reponse. I have created a sample report for testing. Below is the curl command we can use to verify the response:

```
curl -X GET \
https://implementation.concursolutions.com/api/expense/expensereport/v2.0/report/F66E2EC1743944E9929F \
  -H 'accept: application/json' \
  -H 'authorization: OAuth 0_5sfyEKPk8cnYlWaos22OpPsLk='
```